packages/docs-page/server/get-paths-from-nav-data.ts

```diff
@@ -30,7 +30,7 @@ export function getPathsFromNavData(
   const pagePathArrays = getPathArraysFromNodes(navDataResolved)
   // Ensure we include an empty array for the "/" index page path
   // (may be included in nav-data, eg for Terraform, or may not, eg for all other sites)
-  const hasIndexPage = pagePathArrays.filter((p) => p == []).length > 0
+  const hasIndexPage = pagePathArrays.filter((p) => p.length === 0).length > 0
```

Comment:
The two `.length` methods make this a be confusing to read. What do you think about changing it to this:
```suggestion
  const hasIndexPage = pagePathArrays.some((p) => p.length === 0)
```

Reply:
@heatlikeheatwave love it! Plus it exits early as well, which is faster!